ipfx/bin/run_pipeline_from_nwb_file.py

```diff
@@ -18,16 +18,41 @@ def main():
     parser = argparse.ArgumentParser(
         description="Process an nwb file through the ephys pipeline"
     )
-    parser.add_argument("input_nwb_file", type=str)
-    parser.add_argument("output_dir", type=str)
-    parser.add_argument("--write_spikes",
-                        type=bool,
-                        default=False,
-                        help="If true will attempt to append spike times to the nwb file",
-                        )
-    parser.add_argument("--input_json", type=str, default="input_json")
-    parser.add_argument("--output_json", type=str, default="output_json")
-    parser.add_argument("--qc_fig_dir", type=str, default=None)
+    parser.add_argument(
+        "input_nwb_file", type=str, help="process this NWB2 file"
+    )
+    parser.add_argument(
+        "output_dir", type=str, help="outputs will be written here"
+    )
+    parser.add_argument(
+        "--write_spikes", type=bool, default=False,
+        help="If true will attempt to append spike times to the nwb file",
+    )
+    parser.add_argument(
+        "--input_json", type=str, default="input_json",
```

Comment:
it would be better to name it `input.json`- i.e., with an extension. This allows configuring preferred editor for opening files based on extension.

Reply:
Also, check that quick_start page refers to the correct name of the input/output JSON. The last time I saw, it was referring to  `pipeline_input.json` and `pipeline_output.json`

Reply:
good catch, thank you!